script.js

```diff
@@ -1,4 +1,50 @@
+//A basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+
+//The random function to determine the choice for the program
+var randomProgram = function () {
+  var choice = ["scissors", "paper", "stone"];
+  return choice[Math.floor(Math.random() * choice.length)];
+};
+
+//Find the icone for each choice
+var addIcone = function (object) {
+  if (object == "scissors" || object == "reverse scissors") return `✂️`;
+  if (object == "paper" || object == "reverse paper") return `🗒`;
+  if (object == "stone" || object == "reverse stone") return `🪨`;
```

Comment:
I'm not too sure why You have "reverse scissors" and "reverse paper" and "reverse stone". But good effort in returning icons!